### ISSUE TITLE
[request] in v2 docs, it's hard to find the path to specific config field

### ISSUE THREAD

**Issue by thewh1teagle** [tauri-apps/tauri-docs#2225]
### Question you want answered

How do I find the absolute path to config field in `tauri.conf.json`

### Where did you look for an answer?

For instance it's not clear where should I place [config/#nsisconfig](https://v2.tauri.app/reference/config/#nsisconfig) inside `tauri.conf.json`.
I think the docs should somehow describe on each config field what's the absolute config path to that field and where to add it exactly inside `tauri.conf.json`

### Page URL

https://v2.tauri.app/reference/config

### Are you willing to work on this yourself?

- [ ] I want to work on this myself

**Comment by simonhyll** [comment]
I fully agree that the config reference is hard to navigate. I've toyed with a couple different setups and it's hard to improve what we already have in a way that's sort of "definitively an improvement". My current idea is that we split up the config into a bunch of sub pages, takes a bit of effort is all in the config generator.

Tracking for 2.x simply because the current version is usable, just not as good as we could potentially have it.

**Comment by simonhyll** [comment]
Mocked up how the sidebar might be used to make the config easier to navigate, perhaps, possibly

![Image](https://github.com/tauri-apps/tauri-docs/assets/79983560/b6ee7e72-95a9-4bd8-b1af-03c9a0594fd3)

**Comment by thewh1teagle** [comment]
It's not just about navigating the config (although that certainly needs improvement). It's also about the process of eventually finding a specific configuration and then applying it within my own configuration. There needs to be a simple way to understand the **path** for adding that particular configuration. For instance, `app -> nsis -> some -> other -> very -> very -> nested -> config`
